.travis.yml

```diff
@@ -143,3 +149,9 @@ jobs:
             - node
       before_install:
         - export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"
+        - pip install PyGithub  # used to upload packages to Github releases
+      script:
+        - if [[ -z "${TRAVIS_TAG}" ]]; then python src/packaging/set_commit_in_version.py $TRAVIS_COMMIT; fi
+        - make distrib -j2
+#        - if [[ "$(ls -l distribution | grep -v ^l | wc -l)" -ne "3" ]]; then exit -1; fi  # make sure all the required file are generated
```

Comment:
Why did you commented out these lines? We should either remove them or explain why they are commented out.